ouroboros-consensus-diffusion/src/ouroboros-consensus-diffusion/Ouroboros/Consensus/Network/NodeToNode.hs

```diff
@@ -42,7 +42,8 @@ import qualified Codec.CBOR.Decoding as CBOR
 import           Codec.CBOR.Encoding (Encoding)
 import qualified Codec.CBOR.Encoding as CBOR
 import           Codec.CBOR.Read (DeserialiseFailure)
-import           Control.Concurrent.Class.MonadSTM.Strict.TVar as TVar.Unchecked
+import           Control.Concurrent.Class.MonadSTM.Strict.TVar as TVar.Unchecked hiding
+                     (modifyTVar)
```

Comment:
I think this should have been a `qualified` import in the first place, right?

Reply:
Indeed. Fixed in 2d434fed7d5c31f63abe896224a63974782f22ea. It's a bit puzzling to me why there is both STM's TVars and io-classes's TVars in the same project, but that's out of the scope of this PR.

Reply:
Yeah, that's weird, note that it is "normal" TVars from io-classes, and TVars with NoThunks checks from strict-checked-vars.